physionet-django/user/templates/user/edit_training.html

```diff
@@ -50,8 +50,10 @@ <h4>Submit evidence of a completed course</h4>
     {% endif %}
   {% endfor %}
 
-  <p>Please use the form below to submit a new completion report. 
-    For CITI training, please refer to our <a href="{% url "citi_course" %}" target="_blank">step-by-step instructions</a>. 
+  <p>Please use the form below to submit a new completion report.</p>
+  <p>For CITI training, please refer to our <a href="{% url "citi_course" %}" target="_blank">step-by-step instructions</a> and make sure to:
+    <ul>1) Upload the training report and not the training certificate.</ul>
```

Comment:
Here you could change the `<ul>` to `<ol>` and remove the `1)` and `2)`

Reply:
Yes for the ordered list, but since we want to remove the certificate aspect can I leave point 2 and keep it unordered and for point 2 say "if the requesting for MIMIC datasets please complete the Data or Specimens Only Research course" as most people ask for MIMIC and upload Biomedical Investigators or others, so we could clarify it there upon upload of the pdf.

Reply:
Thanks Lama. I think if we are keeping just one point (which sounds good) then perhaps no need to use a list at all? I think it's okay to mention the Data or Specimen's course for now, but I don't think we should discuss MIMIC (which would be confusing on other platforms).

@alistairewj are you okay with this being added to the training page?:

`For CITI training, please refer to our <a href="{% url "citi_course" %}" target="_blank">step-by-step instructions</a> and be sure to <strong>upload the training report<strong>, not the certificate.`

Reply:
Yes I agree I think we should mention “Data or Specimens Only Research” course as people we have any CITI training wouldn't click on the link to read CITI instructions and end uploading any course for CITI. 
 If we say CITI training we should add for them to complete “Data or Specimens Only Research” course with HIPAA module somewhere upon report upload.